README.md

```diff
@@ -13,16 +13,22 @@ layout:
     visible: true
 ---
 
-# Kontrol
-
 <div data-full-width="true">
 
 <figure><img src=".gitbook/assets/kontrol logo yellow.png" alt=""><figcaption></figcaption></figure>
 
 </div>
 
-[**Kontrol**](https://github.com/runtimeverification/kontrol) combines **KEVM** and [Foundry](https://book.getfoundry.sh/) to grant developers the ability to perform [formal verification](https://en.wikipedia.org/wiki/Formal\_verification) without learning a new language or tool. This is especially useful for those who are not verification engineers. Additionally, developers can leverage Foundry test suites they have already developed and use symbolic execution to increase the level of confidence.
+# Kontrol
+
+Struggling to kontrol your smart contracts behave as intended, even after thorough unit testing, fuzzing, and other validation procedures? Frustrated with finding bugs in your code but unable to prove their absence?
```

Comment:
I guess I wouldn't say that people are struggling to ensure their contracts behave as intended until they get hacked.  Maybe we can point out to external manual audits being less reliable and sustainable than automated tooling and CI? I just generally think that people really consider "unit testing, fuzzing, and other validation procedures" sufficient, so maybe we could reformulate it as something along the lines of  "Kontrol provides an easy-to-use yet powerful approach to formal verification, so you don't have to settle for techniques offering less security guarantees like fuzzing and unit testing".

Reply:
I want to express a situation that deeply resonates with developers. I'd prefer not to mention audits since we also provide audit services. What do you think about the following one?

```suggestion
[**Kontrol**](https://github.com/runtimeverification/kontrol) provides an easy-to-use yet powerful formal verification for smart contracts. It proves that smart contracts will execute exactly as you constrain them to, rather than just finding bugs. Formal constraints are provided in a way similar to foundry tests and can be integrated into CI processes through [KaaS](https://docs.runtimeverification.com/kaas). Thus, with minimal effort, it continuously offers higher security guarantees than fuzzing and unit testing.
```

Reply:
Sure, please feel free to run it by him!